model/profiles/frame.yaml

```diff
@@ -0,0 +1,8 @@
+groups:
+  - id: profile.frame
+    type: attribute_group
+    brief: >
+      Describes the origin of a single frame in a Profile.
+    attributes:
+      - ref: profile.frame.type
+        requirement_level: opt_in
```

Comment:
Is it really opt in here? Wouldn't we always want to know the frame type? 🤔

Reply:
The understanding and agreement is, that `profile.frame.type` is optional, as it is "just" required for visualization of the data. Data without `profile.frame.type` can still be processed and provides its value.

Reply:
I see. Maybe then `recommended` is better here? `opt-in` requires the explicit opt-in from the user via configuration/sdk setup whereas recommended is "on" by default, but can be not added if there's good reason not to, like if it's expensive to collect and etc https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/attribute-requirement-level.md#recommended

Reply:
Updated the level to `recommended` with https://github.com/open-telemetry/semantic-conventions/pull/1188/commits/94bb7ed8f9f3288bffa02744058071c3d36ca4af.